setup.py

```diff
@@ -47,13 +52,26 @@ def __init__(self, *args, **kwargs):
 
 setup(
     name='SciencePlots',
```

Comment:
Could you change this to lower case? If we're going to need to use `import SciencePlots` now, I'd prefer it to be lower case (as in `import scienceplots`).